code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -1233,6 +1233,20 @@
 		light(user,I)
 		update_brightness(user)
 		return
+	else if(istype(I, /obj/item/weapon/reagent_containers/food/snacks/donut))
```

Comment:
If you make it possible to dip fries in ketchup dear God make it like 1 unit.

🇧🇪